doc/mm.md

```diff
@@ -52,6 +52,7 @@ where code size matters and you know that your code does not produce cycles, you
 use `--mm:arc`. Notice that the default `async`:idx: implementation produces cycles
 and leaks memory with `--mm:arc`, in other words, for `async` you need to use `--mm:orc`.
 
+Only ARC/ORC support move semantics, destructors, views, `sink`, `lent`, `cursor`, `acyclic` and incremental compilation.
```

Comment:
Well `lent` and `views` is not affected and IC does not work anyway.

Reply:
Any other relevant feature that is not officially supported on refc ? 🤔